analysis/pricesignal.py

```diff
@@ -42,14 +42,27 @@ def getdam(
     return dam_data[dam_data["month"] == month]["USD_per_MWh"].values
 
 def gettariff(
-    region, basepath=os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    region, full_list=False, basepath=os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 ):
-    """Get the tariff for a given region."""
-    # Load the tariff data
-    tariff_path = os.path.join(basepath, "data", "tariff", f"{region}tariff.csv")
-    
-    # read df
-    tariff_data = pd.read_csv(tariff_path)
+    """Get the tariffs for a given region."""
```

Comment:
Is full_list just an identifier to get several tariffs instead of just one?

Reply:
Yeah it loops through and gets all the tariffs for the region instead of just one. I can update the docstring to make that clear